tsl/test/sql/continuous_aggs_policy.sql

```diff
@@ -86,9 +86,9 @@ CREATE MATERIALIZED VIEW max_mat_view_date
         GROUP BY 1 WITH NO DATA;
 
 \set ON_ERROR_STOP 0
-SELECT add_continuous_aggregate_policy('max_mat_view_date', '2 days'::interval, 10 , '1 day'::interval);
+SELECT add_continuous_aggregate_policy('max_mat_view_date', '2 days', 10, '1 day'::interval);
 \set ON_ERROR_STOP 1
-SELECT add_continuous_aggregate_policy('max_mat_view_date', '2 day'::interval, '1 day'::interval , '1 day'::interval) as job_id \gset
+SELECT add_continuous_aggregate_policy('max_mat_view_date', '2 day', '1 day', '1 day'::interval) as job_id \gset
```

Comment:
Nit: Would be nice to get rid of the cast on schedule_interval too. Perhaps also for a separate PR?

```suggestion
SELECT add_continuous_aggregate_policy('max_mat_view_date', '2 days', '1 day', '1 day'::interval) as job_id \gset
```